bundles/AdminBundle/Resources/public/js/pimcore/log/admin.js

```diff
@@ -193,6 +193,7 @@ pimcore.log.admin = Class.create({
                     flex: 220,
                     sortable: true,
                     renderer: function (s) {
+                        meta.tdAttr='data-qtip="'+s+'"';
```

Comment:
What's `meta`? Seems it isn't defined anywhere :confused:

Reply:
It's part of renderer function. It's been almost two months, I made this PR. It was working fine during that time. I have to check again.

May be we need to alter the function as below. 

renderer: function (s, metadata, record) {
                        meta.tdAttr='data-qtip="'+s+'"';
                        return Ext.util.Format.htmlEncode(s);
                    }

Reply:
yeah, seems like the parameter is missing

Reply:
@shamoh19 🏓 can you please recheck? Thanks!